src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -0,0 +1,75 @@
+from numpy import nan
+
+
+class EstimandException(Exception):
+    pass
+
+
+RESULTS_PREFIX = "results_"
+BASELINE_PREFIX = "baseline_"
+
+
+class Estimandizer:
+    """
+    Estimandizer. Generate estimands explicitly.
+    """
+
+    def add_estimand_results(self, data_df, estimands, historical):
+        columns_to_return = []
+        for estimand in estimands:
+            results_col = f"{RESULTS_PREFIX}{estimand}"
+            if results_col not in data_df.columns:
+                # will raise a KeyError if a function with the same name as `estimand` doesn't exist
+                try:
+                    data_df = globals()[estimand](data_df, RESULTS_PREFIX)
+                except KeyError as e:
+                    if historical:
+                        data_df[results_col] = nan
+                    else:
+                        raise e
+            columns_to_return.append(results_col)
+
+        results_column_names = [x for x in data_df.columns if x.startswith(RESULTS_PREFIX)]
+        # If this is not a historical run, then this is a live election
+        # so we are expecting that there will be actual results data
+        if not historical and len(results_column_names) == 0:
```

Comment:
I'm pretty sure this will have failed previously now, right? Since it's the case where historical is false but results col is not in the data?

Reply:
Hmmmmmmm...yes, I think that's right.  This was copied over from `MockLiveDataHandler` but now this same check is being performed in the `for` loop above.  Also I just commented it out and `tox` passed 😄 so I'll go ahead and remove this.  Thanks!